### PR TITLE
Removed EditInPlaceMessage

### PR DESCRIPTION
**Pull request by Nyholm**
We can use Common\Message instead

### CODE REVIEW

File: Controller/EditInPlaceController.php [hunk 1]
```diff
@@ -97,10 +98,11 @@ private function getMessages(Request $request, array $validationGroups = [])
         foreach ($data as $key => $value) {
             list($domain, $translationKey) = explode('|', $key);
 
-            $message = new EditInPlaceMessage();
+            $message = new Message();
             $message->setKey($translationKey);
-            $message->setMessage($value);
+            $message->setTranslation($value);
             $message->setDomain($domain);
+            $message->setDomain($locale);
```
Comment (damienalexandre) [inline Controller/EditInPlaceController.php]: setLocale?

Comment (Nyholm) [inline Controller/EditInPlaceController.php]: Thank you

**Comment by Nyholm** [comment]
Thank you for the review. I will merge #128 first and then make sure to write some tests to this PR to make sure the validation works.

File: Controller/EditInPlaceController.php [hunk 2]
```diff
@@ -97,10 +105,11 @@ private function getMessages(Request $request, array $validationGroups = [])
         foreach ($data as $key => $value) {
             list($domain, $translationKey) = explode('|', $key);
 
-            $message = new EditInPlaceMessage();
+            $message = new Message();
```
Comment (rvanlaak) [inline Controller/EditInPlaceController.php]: Based on this PR it seems the common `Message` can be made immutable (by removing setters and adding constructor arguments). Am I right about this?

Comment (Nyholm) [inline Controller/EditInPlaceController.php]: Yeah, for this use case.. But do we want it immutable? hm, maybe we do.

**Pull request merged**